targets/TARGET_STM/TARGET_STM32F4/STM32Cube_FW/system_stm32f4xx.c

```diff
@@ -166,7 +166,8 @@ const uint8_t APBPrescTable[8]  = {0, 0, 0, 0, 1, 2, 3, 4};
   * @param  None
   * @retval None
   */
-void SystemInit(void)
+//__weak void SystemInit(void) FONCTIONNE
```

Comment:
Remove this line ?

Reply:
I started CI and it's marked now as ready for merge, should this be removed then restarted CI?

Reply:
OK